<robot>
  <link name="finger_holder">

    <inertial>

      <origin rpy="0 0 0" xyz="-0.012491 0.037779 -0.056801"/>
      <!--<origin rpy="0 0 0" xyz="-0.012491 0.051931 -0.05963"/>-->

      <mass value="0.068815"/>
      <!--<mass value="0.075511"/>-->

      <inertia ixx="0.0002228" ixy="4.3122e-8" ixz="-2e-12" iyy="0.0001279" iyz="0.00005549" izz="0.0001886"/>
      <!--<inertia ixx="0.0003859" ixy="4.3121e-8" ixz="-2e-12" iyy="0.0001343" iyz="0.00008656" izz="0.0003455"/>-->

    </inertial>

    <visual>

      <origin rpy="0 0 0" xyz="0 0 0"/>

      <geometry>

        <mesh filename="package://kinova_mujoco/meshes/finger_holder.STL" scale="0.001 0.001 0.001"/>
      </geometry>

      <material name="DUMMY_MATERIAL">

        <color rgba="1 1 1 1"/>

      </material>

    </visual>

    <collision>

      <origin rpy="0 0 0" xyz="0 0 0"/>

      <geometry>

        <mesh filename="package://kinova_mujoco/meshes/finger_holder.STL" scale="0.001 0.001 0.001"/>
      </geometry>

    </collision>

  </link>

  <joint name="finger_holder_base_gripper" type="fixed">

    <parent link="robotiq_arg2f_base_link"/>

    <child link="finger_holder"/>

    <!--<origin rpy="0 0 0" xyz="-0.0325 -0.0125 0.003"/>-->
    <origin rpy="0 0 3.14" xyz="0.0325 0.0125 0.003"/>

  </joint>

  <link name="finger_link"/>
  <joint name="finger_base_gripper" type="fixed">

    <parent link="robotiq_arg2f_base_link"/>

    <child link="finger_link"/>
    <!--0.175 0.1775-->
    <origin rpy="0.0 0 0.0" xyz="-0.05642 0 0.1775"/>
    <!--<origin rpy="0.0 0 0.0" xyz="0.05642 0 0.175"/>-->
  </joint>

</robot>